.gitignore

```diff
@@ -10,4 +10,5 @@ test-results/
 playwright-report/
 coverage/
 public/uploads/*
-.vscode/*
\ No newline at end of file
+.vscode/*
+node_modules
```

Comment:
I think this is already covered by the first line https://github.com/dev-launchers/strapiv4/pull/250/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R1.